testapp/__main__.py

```diff
@@ -0,0 +1,50 @@
+import sys
```

Comment:
Beats the no test at all that we had before, but indeed unit tests would be lovely.